swarm/tools/runs_gc.py

```
@@ -73,16 +74,23 @@ def age_days(self) -> float:
+    while stack:
+        current = stack.pop()
+        try:
+            with os.scandir(current) as it:
+                for entry in it:
+                    if entry.is_dir(follow_symlinks=False):
+                        stack.append(entry.path)
+                    elif entry.is_file():
+                        try:
+                            total += entry.stat().st_size
+                        except OSError:
+                            pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)